.github/workflows/assign.yml

```diff
@@ -0,0 +1,16 @@
+name: Assign members based on comments
```

Comment:
Why are there two workflow files?

Reply:
`.github/workflows/assign.yml ` defines the workflow to use the Github Action in the project. The other one (`.github/actions/assign-on-ptal/action.yml `) defines the Action and how it is executed.